linux_os/guide/system/bootloader-grub2/grub2_password/oval/shared.xml

```diff
@@ -1,3 +1,4 @@
+{{% set grub_cfg_prefix = "/boot/grub2" %}}
```

Comment:
What is this good for?

Reply:
Basically one can diff the uefi and non-uefi version, and the diff is very small.

Reply:
aha, that's clever